src/game/shared/schemainitutils.h

```diff
@@ -25,14 +25,17 @@
 		}																\
 		else															\
 		{																\
-#ifndef VALVE_PURE														\
+/* fixme(replaycoding):
```

Comment:
Could you fix this?

Reply:
Yes... Is what I would say if this actually changed anything, reverting this will still build on sniper. :woman_shrugging: (maybe it failed to build on soldier & scout?)

Reply:
According to https://stackoverflow.com/questions/10074520/why-compiler-complain-about-this-macro-declaration, this isn't actually possible to fix without moving VALVE_PURE to outside the macro (ew, we're not doing that), so i'm just going to leave it commented out, with a different comment.